Site/webapp/wdkCustomization/js/client/components/Srt.tsx

```diff
@@ -106,20 +106,15 @@ const SUPPORTED_RECORD_CLASS_CONFIGS: InitialSrtFormConfig[] = [
   }
 ];
 
-const IDS_HELP_TOOLTIP_POSITION = {
-  my: 'bottom left',
-  at: 'top right'
-};
-
 export function Srt() {
   const compatibleSrtConfigs = useCompatibleSrtFormConfigs();
-  const [ selectedSrtForm, setSelectedSrtForm ] = useState<string>('');
+  const [selectedSrtForm, setSelectedSrtForm] = useState<string>('');
```

Comment:
Hi Jeremy - I just had a quick look at https://github.com/VEuPathDB/web-components/pull/335 and ended up here and see that a lot of code formatting changes have been committed in this PR.  I am guessing that this repository doesn't have the peri-commit prettification actions that we have in web-eda and web-components. I'm not sure what the solution is - other than to revert the changes and set up your IDE to make no style/formatting changes?  Best talk to @jtlong3rd I think!